models/nft/arbitrum/nft_arbitrum_schema.yml

```diff
@@ -179,4 +179,19 @@ models:
       - name: tx_to
         description: "Address that the transaction points to"
       - name: evt_index
-        description: "Event index"
\ No newline at end of file
+        description: "Event index"
+
+  - name: nft_arbitrum_aggregators
```

Comment:
maybe add unique test on address here? i didn't check other blockchains, but if not there either, maybe worthwhile to add. will help avoid unnecessary dupes downstream in nft trades

Reply:
done ✅